src/utils/RacingCarUtils.js

```diff
@@ -0,0 +1,53 @@
+import {Console, Random} from '@woowacourse/mission-utils';
+import RacingCar from "../RacingCar.js";
+import ValidationUtils from "./ValidationUtils.js";
+import ErrorCode from "../datas/ErrorCode.js";
+
+class RacingCarUtils {
+    static findWinners(carList) {
+        if (carList.length <= 0) throw new Error(ErrorCode.CAR_NOT_ENOUGH)
+
+        carList.sort((a, b) => b.getDistance() - a.getDistance())
+
+        const maxDist = carList[0].getDistance();
+        return carList.filter(car => car.getDistance() === maxDist)
+            .map(car => car.name);
+    }
+
+    static createCarListByPlayerList(players) {
+        if (ValidationUtils.checkDuplicate(players)) {
+            throw new Error(ErrorCode.CAR_NAME_DUPLICATE)
+        }
+        return players.reduce((cars, player) => {
+            cars.push(new RacingCar(player));
+            return cars;
+        }, []);
+    }
+
+    static race(carList, tryNumber) {
+        let num = ValidationUtils.isValidNumber(tryNumber);
+        while (num--) {
+            carList.forEach(car => {
+                car.goForward(Random.pickNumberInRange(0, 9))
+                car.printDistance(car)
+            })
+            Console.print('') //개행
+        }
+        return carList;
+    }
```

Comment:
이부분도 구조를 크게 변경하지 않고 const로 작성하는 예시 중 하나를 가져와봤어요
물론 개인 및 조직 별 컨벤션에 따라 다르겠지만, `while`은 루프 종료 조건이 직관적이지 않은 경우가 많고 종료 조건이 불명확한 경우가 종종 있어서 대부분 `for` 구문을 더 선호하는 것 같아요!

```suggestion
    static race(carList, tryNumber) {
        const num = ValidationUtils.isValidNumber(tryNumber);

        for (let i = 0; i < num; i++) {
            carList.forEach(car => {
                car.goForward(Random.pickNumberInRange(0, 9));
                car.printDistance(car);
            });
            Console.print(''); // 개행
        }

        return carList;
    }
```

Reply:
num을 변경하지 않고도 다음과 같은 방법으로 해결이 가능하군요!
위에 남겨주신 레퍼런스 꼭 숙지해서 다음부터는 let사용을 최소화하겠습니다
감사합니다!